rest/taskana-rest-spring/src/test/java/pro/taskana/rest/AccessIdControllerIntTest.java

```diff
@@ -33,48 +35,52 @@ static void init() {
 
   @Test
   void testQueryGroupsByDn() {
-    ResponseEntity<List<AccessIdResource>> response =
+    ResponseEntity<AccessIdListResource> response =
         template.exchange(
             restHelper.toUrl(Mapping.URL_ACCESSID)
-                + "?search-for=cn=developersgroup,ou=groups,o=taskanatest",
+                + "?search-for=cn=ksc-users,cn=groups,OU=Test,O=TASKANA",
             HttpMethod.GET,
             restHelper.defaultRequest(),
-            ParameterizedTypeReference.forType(List.class));
+            ParameterizedTypeReference.forType(AccessIdListResource.class));
     assertThat(response.getBody()).hasSize(1);
+    assertThat(response.getBody().get(0).getAccessId())
+        .isEqualToIgnoringCase("cn=ksc-users,cn=groups,OU=Test,O=TASKANA");
   }
 
   @Test
   void testQueryGroupsByCn() {
-    ResponseEntity<List<AccessIdResource>> response =
+    ResponseEntity<AccessIdListResource> response =
         template.exchange(
-            restHelper.toUrl(Mapping.URL_ACCESSID) + "?search-for=developer",
+            restHelper.toUrl(Mapping.URL_ACCESSID) + "?search-for=ksc",
             HttpMethod.GET,
             restHelper.defaultRequest(),
-            ParameterizedTypeReference.forType(List.class));
+            ParameterizedTypeReference.forType(AccessIdListResource.class));
     assertThat(response.getBody()).hasSize(1);
+    assertThat(response.getBody().get(0).getAccessId())
+        .isEqualToIgnoringCase("cn=ksc-users,cn=groups,OU=Test,O=TASKANA");
   }
 
   @Test
   void testGetMatches() {
     ResponseEntity<List<AccessIdResource>> response =
         template.exchange(
-            restHelper.toUrl(Mapping.URL_ACCESSID) + "?search-for=ali",
+            restHelper.toUrl(Mapping.URL_ACCESSID) + "?search-for=rig",
             HttpMethod.GET,
             restHelper.defaultRequest(),
             ParameterizedTypeReference.forType(AccessIdListResource.class));
 
     List<AccessIdResource> body = response.getBody();
     assertThat(body).isNotNull();
-    assertThat(body).hasSize(3);
+    assertThat(body).hasSize(2);
     assertThat(body)
         .extracting(AccessIdResource::getName)
-        .containsExactlyInAnyOrder("Tralisch, Thea", "Bert, Ali", "Mente, Ali");
+        .containsExactlyInAnyOrder("Schläfrig, Tim", "Eifrig, Elena");
```

Comment:
can we use the accessId instead? Personally I think people tend to change names more often than ids